frontend/javascripts/admin/voxelytics/workflow_list_view.tsx

```diff
@@ -17,28 +17,26 @@ import { VX_POLLING_INTERVAL } from "./workflow_view";
 function parseTaskInfo(taskInfo: VoxelyticsTaskInfo): VoxelyticsTaskInfo {
   return {
     ...taskInfo,
-    beginTime: taskInfo.beginTime != null ? new Date(taskInfo.beginTime) : null,
-    endTime: taskInfo.endTime != null ? new Date(taskInfo.endTime) : null,
+    beginTime: taskInfo.beginTime != null ? taskInfo.beginTime : null,
+    endTime: taskInfo.endTime != null ? taskInfo.endTime : null,
```

Comment:
```suggestion
```

See my other comment for an explanation. Also here the short form if deleting doesnt work: `taskInfo.beginTime ?? null`